html-links/index.html

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>links</title>
+</head>
+</html>
```

Comment:
All your elements should be children of the `<html>` tag.  Your `<body>` fell out.